packages/dds/matrix/src/serialization.ts

```diff
@@ -3,24 +3,17 @@
  * Licensed under the MIT License.
  */
 
-import {
-    Serializable,
-    IFluidDataStoreRuntime,
-    IChannelStorageService,
-} from "@fluidframework/datastore-definitions";
+import { Serializable, IChannelStorageService } from "@fluidframework/datastore-definitions";
 import { FileMode, TreeEntry } from "@fluidframework/protocol-definitions";
-import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { IFluidHandle, IFluidSerializer } from "@fluidframework/core-interfaces";
 import { fromBase64ToUtf8 } from "@fluidframework/common-utils";
 
-export function serializeBlob(
-    runtime: IFluidDataStoreRuntime,
+export const serializeBlob = (
     handle: IFluidHandle,
     path: string,
     snapshot: Serializable,
-) {
-    const serializer = runtime.IFluidSerializer;
-
-    return {
+    serializer?: IFluidSerializer,
```

Comment:
I wonder if this argument needs to be required. Is the scenario where it's optional - DDS that does not handle handles?
If yes, should it be even calling into serializeBlob? It feels like if DDS knows for sure it does not support handles, then it should not call this API. Otherwise it should always have serializer.
What do you think?

Reply:
Looking at the implementation, it looks like the expectation is that it can serialize data without handles too.
However, I don't see anyone not passing in a serializer today so I am not sure what the use case is. Maybe @DLehenbauer would know?

Reply:
Haha...  So, this whole thing with 'runtime.IFluidSerializer' potentially being undefined was a pattern that I think we had in all our DDSes at some point?

I was just about to copy/paste it again for SharedLog/Tree and thinking "I bet this is left over cruft from when handles were first introduced.  I should ask Navin/Vlad if I still need to support this..."

So yeah...  SharedMatrix always supports handles and I'd love for the argument to be non-optional.  :-)

Reply:
Awesome! I will remove it then :-).